static/docs/2.x/modules.svg

```diff
@@ -0,0 +1,3 @@
on </span><br style="font-size: 11px" /><font color="#2f495e" style="font-size: 11px"><span style="font-size: 11px">    function () {} </span><br style="font-size: 11px" /></font></div><div style="font-size: 11px">  ]</div><div style="font-size: 11px">}</div></div></div></div></foreignObject><text x="401" y="133" fill="#808080" font-family="Courier New" font-size="11px" font-weight="bold">// nuxt.config.js...</text></switch></g><rect x="713" y="238" width="124.5" height="424" fill="none" stroke="none" pointer-events="all"/><path d="M 837.5 238 L 775.25 238 L 775.25 662 L 837.5 662" fill="none" stroke="#2f495e" stroke-width="4" stroke-miterlimit="10" pointer-events="all"/><path d="M 713 450 L 775.25 450" fill="none" stroke="#2f495e" stroke-width="4" stroke-miterlimit="10" pointer-events="all"/><rect x="1344" y="40" width="274" height="98" fill="none" stroke="none" pointer-events="all"/><g transform="translate(-0.5 -0.5)scale(2)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe flex-start; justify-content: unsafe flex-end; width: 135px; height: 1px; padding-top: 27px; margin-left: 672px;"><div style="box-sizing: border-box; font-size: 0; text-align: right; "><div style="display: inline-block; font-size: 15px; font-family: Helvetica; color: #2F495E; line-height: 1.2; pointer-events: all; font-weight: bold; white-space: normal; word-wrap: normal; ">Nuxt v2.14<br style="font-size: 15px" />September 2020</div></div></div></foreignObject><text x="807" y="42" fill="#2F495E" font-family="Helvetica" font-size="15px" text-anchor="end" font-weight="bold">Nuxt v2.14...</text></switch></g><rect x="84" y="2110" width="612" height="144" fill="none" stroke="none" pointer-events="all"/><g transform="translate(-0.5 -0.5)scale(2)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe flex-start; justify-content: unsafe flex-start; width: 304px; height: 1px; padding-top: 1062px; margin-left: 44px;"><div style="box-sizing: border-box; font-size: 0; text-align: left; "><div style="display: inline-block; font-size: 14px; font-family: Courier New; color: #808080; line-height: 1.2; pointer-events: all; font-weight: bold; white-space: normal; word-wrap: normal; ">// nuxt.config.js<br style="font-size: 14px" />export default { <br style="font-size: 14px" /> <font color="#2f495e" style="font-size: 14px">modules:</font> ['@nuxtjs/sitemap']<br style="font-size: 14px" />}</div></div></div></foreignObject><text x="44" y="1076" fill="#808080" font-family="Courier New" font-size="14px" font-weight="bold">// nuxt.config.js...</text></switch></g></g></svg>
```

Comment:
Looking great. Spotted a few tiny typos in the modules SVG:
* `modules are Nuxt.js extensions that extend`
* `create async modules that return a promise`
* `simply, add dependency in devDependencies and use buildModules`

Reply:
Thanks @danielroe All typos are fixed.